src/main/java/net/dries007/tfc/types/DefaultRecipes.java

```diff
@@ -134,10 +138,168 @@ public static void onRegisterBarrelRecipeEvent(RegistryEvent.Register<BarrelReci
             // this ratio works for 9b + 1b = 10b (full barrel) of brine/milk_vinegar, but leaves odd ninths of fluid around for other mixtures.
             //  previous todo had "make it a simpler calculation"
             new BarrelRecipe(IIngredient.of(FRESH_WATER.get(), 500), IIngredient.of("dustFlux"), new FluidStack(LIMEWATER.get(), 500), ItemStack.EMPTY, 0).setRegistryName("limewater"),
+            new BarrelRecipe(IIngredient.of(LIMEWATER.get(), 100), IIngredient.of(ItemsTFC.GYPSUM), null, new ItemStack(BlocksTFC.ALABASTER_RAW_PLAIN), ICalendar.TICKS_IN_HOUR).setRegistryName("plain_alabaster"),
 
             new BarrelRecipeTemperature(IIngredient.of(FRESH_WATER.get(), 1), 50).setRegistryName("fresh_water_cooling"),
             new BarrelRecipeTemperature(IIngredient.of(SALT_WATER.get(), 1), 50).setRegistryName("salt_water_cooling")
         );
+
+        //Nice handy list of dye in the correct order BY META DATA to get the correct fluid.
+        List<FluidWrapper> fluids = new ArrayList<>();
```

Comment:
No please. If the fluids are stored via `Map<EnumDyeColor, FluidWrapper>`, the meta data can be retrieved automatically via `EnumDyeColor#getMetadata()`

i.e. `fluids.get(dyeMeta).get()` gets replaced wth `FluidsTFC.DYE.get(color).get()`